python-threatexchange/threatexchange/cli/tests/hash_cmd_test.py

```diff
@@ -46,11 +46,12 @@ def test_stdin(
 ):
     tmp_file.write_text("http://evil.com")
 
-    monkeypatch.setattr("sys.stdin", tmp_file.open())
-    hash_cli.assert_cli_output(
-        ("url", "-"),
-        "url_md5 6d3af727a4e7b025fd59a5469b3a9c57",
-    )
+    with tmp_file.open() as fake_stin:
+        monkeypatch.setattr("sys.stdin", fake_stin)
+        hash_cli.assert_cli_output(
+            ("url", "-"),
+            "url_md5 6d3af727a4e7b025fd59a5469b3a9c57",
+        )
```

Comment:
I didn't do this change. Probably a rebase thing..

Either way, once #1157 get merged, I'll rebase this again and figure out what to do here.

Reply:
Yeah, commented on another line.